scripts/pre_commit_linter.py

```diff
@@ -160,6 +160,10 @@
     }
 }
 
+ALLOWED_TERMINATING_PUNCTUATIONS = ['.', '?']
```

Comment:
Please add back `]` and `}` I had overlooked the cases where there were code samples in the comments (sorry!).

Reply:
That is totally fine. I have fixed this :smile: